jenkins-integration/beaker/install/shared/80_configure_java_args.rb

```diff
@@ -1,38 +1,28 @@
 require 'puppet/gatling/config'
-require 'inifile'
 
 test_name 'Configure java args'
 
 def service_config_name(service_name)
   "/etc/sysconfig/#{service_name}"
 end
 
-def get_service_config(host, service_name)
-  local_service_config_dir = Dir.mktmpdir
-  scp_from(host, service_config_name(service_name), local_service_config_dir)
-  File.join(local_service_config_dir, service_name)
-end
-
-def update_java_args_in_local_config(java_args, local_service_config_path)
-  # Replace the existing JAVA_ARGS in the config
-  ini_file = IniFile.load(local_service_config_path)
-  ini_file['global']['JAVA_ARGS'] = java_args
-  ini_file.save
+def set_bash_variable(host, filepath, variable_name, value)
+  manifest = <<-MANIFEST
+ini_setting { "#{variable_name}":
```

Comment:
One thing that's maybe a bit tricky about this is that I'm not sure for an arbitrary scenario that you're guaranteed to have had `puppetlabs-inifile` installed into the Puppet code directory.  This probably works when we use the `puppetlabs-puppetserver_perf_control` repo for a simulation because its Puppetfile includes `puppetlabs-inifile`.

It might be better to do something like what I did with configuring the `epel` repo [here](https://github.com/puppetlabs/gatling-puppet-load-test/blob/8c5c5a6b557e96a88df375510017e739abfbc2a7/jenkins-integration/beaker/install/shared/install_deps.rb#L2-L5), where we install the module to a temp location (which doesn't potentially pollute the Puppet code dir for the scenario under test) before doing the `puppet apply`.